src/Apps/Sell/sellRoutes.tsx

```diff
@@ -579,8 +579,7 @@ export const sellRoutes: RouteProps[] = [
             },
           },
           {
-            path: "inquiry/:recipientEmail?",
-            getComponent: () => ConsignmentInquiryContainer,
+            path: ":recipientEmail?",
```

Comment:
This fixes the route nesting, as well as the visual nesting (the double logo you can see in the screen cap above)

| Before | After |
|--------|--------|
| ![Screenshot 2024-11-19 at 4 34 29 PM](https://github.com/user-attachments/assets/c340c1ee-4a7d-449e-bfe8-63f20edadc04) | ![Screenshot 2024-11-19 at 4 34 12 PM](https://github.com/user-attachments/assets/aa1cb51a-7b7a-4c12-b539-c64bab2f1ebe) |